assets/js/components/SaptuneDetails/SaptuneDetails.test.jsx

```diff
@@ -14,42 +14,43 @@ import {
 import SaptuneDetails from './SaptuneDetails';
 
 describe('SaptuneDetails', () => {
-  it('should render saptune details correctly', () => {
-    const customSaptuneService = {
-      active: 'active',
-      enabled: 'enabled',
-      name: 'saptune',
-    };
-
-    const customSapconfService = {
-      active: 'inactive',
-      enabled: 'disabled',
-      name: 'sapconf',
-    };
-
-    const customTunedService = {
-      active: null,
-      enabled: null,
-      name: 'tuned',
-    };
-
-    const services = [
-      customSaptuneService,
-      customSapconfService,
-      customTunedService,
-    ];
-    const solution = saptuneSolutionFactory.build({ partial: false });
-    const { notes } = solution;
-    const allNotes = notes.map((id) => saptuneNoteFactory.build({ id }));
-
-    const staging = saptuneStagingFactory.build({ enabled: true });
+  const customSaptuneService = {
+    active: 'active',
+    enabled: 'enabled',
+    name: 'saptune',
+  };
+
+  const customSapconfService = {
+    active: 'active',
+    enabled: 'disabled',
+    name: 'sapconf',
+  };
+
+  const customTunedService = {
+    active: null,
+    enabled: null,
+    name: 'tuned',
+  };
+
+  const services = [
+    customSaptuneService,
+    customSapconfService,
+    customTunedService,
+  ];
 
+  const solution = saptuneSolutionFactory.build({ partial: false });
```

Comment:
Why did you move these constants out?
I think they belong to the previous test.

For the 2nd test, you could use factory values:
```
const { configured_version: configuredVersion, enabled_solution: enabledSolution, applied_solution: appliedSolution } =
      saptuneStatusFactory.build();
```

I don't see having global constants as good idea here

Reply:
fair point i wanted to reuse the same data for the tests, but actually this was the moment where i should have just created a new component :D